scss/_patterns_lists.scss

```diff
@@ -7,6 +7,49 @@ $spv-list-item--inner: null;
   $spv-list-item--inner: 0;
 }
 
+@mixin vf-p-list-placeholders {
```

Comment:
Why have it as a separate mixin and not global?

Reply:
I see it as a convention in all Vanilla scss file, that nothing is added without a mixin.

While placeholders don't add to the size of CSS if they are not used (so potentially they can be global and included automatically with import) they also can duplicate - when same file would be imported multiple times, any global placeholders will also multiply, which can lead to unexpected increase of file size. Keeping them in placeholder and explicitly including them makes it more controlled, so developer can make sure they are included only once.